<?xml version="1.0"?>
<launch>
	<node pkg="pcv_base" type="pcv_base_node" name="pcv_base_node" launch-prefix="sudo -E" output="screen"/>
</launch>
